app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -16,17 +16,10 @@ class SubmissionJob
     end
 
     def retry_limits_for_notification
-      return [1, 10] if Flipper.enabled?(:caregiver1010)
-
-      [10]
+      [1, 10]
     end
```

Comment:
🤔 now that this is reduced to a fixed constant, would it make sense to make it into one?
```suggestion
    RETRY_LIMITS_FOR_NOTIFICATION = [1, 10]
```

Reply:
wait, never mind, this is part of the signature.